### ISSUE TITLE
Plugin prevents password update

### ISSUE THREAD

**Issue by kerstyn** [C-Lodder/Profile-Edit-Redirect#1]
@C-Lodder Unfortunately, I'm still having problems with the plugin. The problem is reproducible on three different devices (chromebook, android tablet and phone). When the plugin is enabled I receive the message (but only on the chromebook) that the passwords don't match and I can't escape from this page even if I type in the homepage url. Without the plugin everything works fine. Also, without the plugin the cancel button works and takes me back to the homepage with a new login menu. With the plugin the cancel button behaves as if it is disabled.

The problem is reproducible even if the browser is set not to store passwords.

**Comment by C-Lodder** [comment]
@kerstyn - On the "Edit Profile" page, did you set in your password in the:
- password field
- confirm password field

??

Also, are you using Joomla 3.4.5?

**Comment by kerstyn** [comment]
Yes to both questions and at no point am I seeing any evidence of auto-filling
of either password fields.
K

**Comment by C-Lodder** [comment]
@kerstyn - Ok thanks. I'll look into this over the weekend

**Comment by kerstyn** [comment]
Merry Christmas. Did you ever manage to figure out what was wrong with that plugin?

**Comment by C-Lodder** [comment]
@kerstyn - Merry Christmas. 

Unfortunately not. I finished work on 24th and needed some time to relax without looking at any code. I'm also on holiday until 11th January, so maybe if you posted on Stackoverflow or Joomla StackExchange, you may get an answer to the problem earlier.

If you do post a question on either of these sites, copy and paste the code from: https://github.com/C-Lodder/Profile-Edit-Redirect/blob/master/profileredirect.php

Removed the copyright comments at the top and say you have written the code.

This way, they won't ask you to "contact the developer" ;)

Else I will have a look when I'm back.

**Comment by kerstyn** [comment]
Not urgent. Enjoy your holiday. I've sent you a connection request on linkedin.
